pretixscan/app/src/main/java/eu/pretix/pretixscan/droid/ui/SetupActivity.kt

```diff
@@ -128,6 +133,25 @@ class SetupActivity : AppCompatActivity(), ScannerView.ResultHandler {
         handleScan(rawResult.text)
     }
 
+    override fun dispatchKeyEvent(event: KeyEvent): Boolean {
```

Comment:
Is this intentionally a very different implementation than in MainActivity? Should we settle on one and maybe even move it to libpretixui?

Reply:
Hmm, have they any differences currently?